group2_assignment2_solution.ipynb

```diff
@@ -509,16 +512,39 @@
    },
    "outputs": [],
    "source": [
+    "k1_filename = 'group2_computer.json'\n",
+    "k2_filename = 'group2_animal.json'\n",
+    "k3_filename = 'group2_movie.json'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'k1_tweets' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-14-a2b69b4a701f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[0mk2_filename\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'group2_animal.json'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0mk3_filename\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'group2_movie.json'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 11\u001b[0;31m \u001b[0msave_to_json\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mk1_tweets\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mk1_filename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     12\u001b[0m \u001b[0msave_to_json\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mk2_tweets\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mk2_filename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     13\u001b[0m \u001b[0msave_to_json\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mk3_tweets\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mk3_filename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mNameError\u001b[0m: name 'k1_tweets' is not defined"
```

Comment:
What happened here?

Reply:
Hmm, I suspect this may just have thrown an error on running a later cell before I started again from to the beginning.

Reply:
It would be cool if there was a button to run all the cells in a notebook. I can't say I've looked for one though 😅

Reply:
I think we should just rerun the whole thing cleanly before submitting just to make sure we dont hit any of these haha

Reply:
Would be easier if we just wrote unit tests :trollface: